XboxKeyboardMouse/Forms/Options.cs

```diff
@@ -732,6 +737,31 @@ private void file_DeletePreset_Click(object sender, EventArgs e)
         // -------------
         #endregion
 
+        private void PrepareInfoTab()
+        {
+            var resources = new System.ComponentModel.ComponentResourceManager(typeof(Options));
+            StringBuilder fullInfo = new StringBuilder();
+
+            fullInfo.AppendLine("XboxKeyboardMouse");
```

Comment:
Static text doesn't need to be rendered dynamically

Reply:
What do you mean?  This was just what I figured was MVP to implement #101 - The version info parts are inserted to the StringBuilder dynamically below.  Or do you mean we should hard-code the version number to the text resource, as another place we have to remember to update it?

There's a number of potential future improvements here.  There could be a separate "about" or version page / dialog if we wanted.  It could be in separate Label elements on the same page, but to do that without impacting the scrolling area would be non-trivial.  Also I just logged some potential future idea as #150.

Reply:
`XboxKeyboardMouse` text is being rendered dynamically when it doesn't need too.  We only need the status.  I don't mind if we leave this code in because of the performance impact in minimal and the feature is more important.